static/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "brid.gy",
```

Comment:
Bridgy

Reply:
manifest needs a name

Reply:
Sure, I meant to change it to `"Bridgy"`, not remove it

Reply:
:D